### ISSUE TITLE
Twitch clips no longer working

### ISSUE THREAD

**Issue by Lumi-s** [extesy/hoverzoom#482]
**Description of the problem**
It seems that Hoverzoom no longer functions with twitch clips as hovering over a link to a clip will only show an image of twitch's logo

**Url of the page where the problem occurs (not the image url)**
clips.twitch.tv I guess?

**Steps to reproduce the problem (include image url here)**
Browsing reddit or another site and hovering over a twitch clip.

OS version, extension version (Chrome only, Firefox is not supported)
Windows 10 64 bit
Extension Version 1.0.138

**Comment by Lumi92** [comment]
Is there any update on this? I'm not proficient enough to help fix it myself.

**Comment by PaulDoesDev** [comment]
I'm also seeing this

**Comment by EhsanKia** [comment]
The original reason why this broke is because the current implementation uses Kraken API which is dead. Helix API is annoying as it requires OAuth. I noticed Imagus sieve uses Twitch's internal GQL API, and I was mostly done implementing that, it all works for fetching the video source, but then noticed that Twitch has added new protection very recently where the clip mp4 won't even load without the right authorization keys. I confirmed that the Imagus sieve no longer works either. So I guess this is dead again until someone finds a way to bypass that.
